dom.bs

```diff
@@ -6939,12 +6982,30 @@ method steps are:
  and null otherwise.
  <!-- This is step 2 of "get an attribute by name", modified as appropriate -->
 
- <li><p>If <var>attribute</var> is null, create an <a>attribute</a> whose
- <a for=Attr>local name</a> is <var>qualifiedName</var>, <a for=Attr>value</a> is
- <var>value</var>, and <a for=Node>node document</a> is <a>this</a>'s <a for=Node>node document</a>,
- then <a lt="append an attribute">append</a> this <a>attribute</a> to <a>this</a>, and then return.
+ <li><p>Let <var>attributeExists</var> be false if <var>attribute</var> is null; otherwise true.
+
+ <li><p>If <var>attributeExists</var> is false, then set <var>attribute</var> to an <a>attribute</a>
+ whose <a for=Attr>local name</a> is <var>qualifiedName</var>, <a for=Attr>value</a> is
+ <var>value</var>, and <a for=Node>node document</a> is <a>this</a>'s <a for=Node>node document</a>.
+
+ <li><p>Let <var>verifiedValue</var> be the result of
+ <a lt="verify attribute value">verifying an attribute value</a> given <var>value</var>,
+ <var>attribute</var>, and <a>this</a>.
+
+ <li><p>Set <var>attributeExists</var> to true if <a>this</a>
+ <a lt="has an attribute">has an attribute</a> <var>attribute</var>; otherwise false.
```

Comment:
As I said on Matrix, I wish we wouldn't need to do identity checks on attributes. That would be a bit tricky, or at least slow in Gecko, and I think also in other engines.

Reply:
per matrix discussion I've updated this and the setAttributeNS equivalent.